runtime/compiler/compile/J9Compilation.cpp

```diff
@@ -171,6 +171,10 @@ J9::Compilation::Compilation(int32_t id,
    _profileInfo(NULL),
    _skippedJProfilingBlock(false),
    _reloRuntime(reloRuntime),
+#if defined(JITSERVER_SUPPORT)
```

Comment:
I am confused about this code. It's part of one of Annabelle's commits and I see the code in the master branch. Why don't I see this code when doing the comparison in this PR?

Reply:
I think you merged Annabelle's commit and if i do a rebase it will resolve the issue. But technically it should tell me that there's a conflict. Let me rebase and see what happens.

Reply:
Okay so technically there is a conflict, but since my change and Annabelle's change are exactly the same, git knows how to auto merge it so it doesn't tell you that there is a conflict. Now after my rebase, those changes are gone.